### PR TITLE
Add timeout(-t) option

### LINKED ISSUE #168: `varnishreload` fails for non-trivial (large) VCLs
Since Varnish 6.6/Jammy (official .deb) (for some reason, it did not occur on 6.5/Bionic (Packagecloud)), recompiling my VCL takes a non-trivial amount of time (about 10 seconds). This may be because I have a large ACL weighing in at 165 KB. This causes `varnishreload` to fail because it uses two discrete invocations of `varnishadm` to both compile (`vcl.load`) and activate (`vcl.use`) the VCL. Although it is possible to work around this issue by specifying an arbitrary wait (`-w 10`), it should be considered a poor solution for two reasons:

1. The user should have the reasonable expectation that simply running `varnishreload` will reload their configuration (without the need to know of or the use of any command options).
2. Arbitrary waits are a fundamentally flawed approach because we either wait too long (to be safe) or too little (causing a failure).

I would like to propose a better solution: send all commands to `varnishadm` together. This works because the invocation of `vcl.load` is inherently blocking, which means queuing up the `vcl.use` after it will only wait the exact amount of time required before the next invocation of `vcl.use` is safe to follow, which seems to me to be a much more elegant solution. The reason the current script fails is because discrete invocations of `varnishadm` are independent and do not know of or care about any pending commands in other processes.

The following proof of concept is all that is required to reload my configuration successfully, demonstrating the point.

```bash
#!/bin/bash
name=$(date +'test-%s')

varnishadm <<-.
	vcl.load "$name" /etc/varnish/default.vcl
	vcl.use "$name"
.
```

Might I suggest integrating this approach, and at the same time, dropping the unnecessary `-w` option?

### PR DESCRIPTION
**Pull request by xcir**
fix #168 

```
# default (vcl is large)
ubuntu@proxy04:~$ sudo ./varnishreload      
Command: varnishadm  -n '' -- vcl.load reload_20240709_143005_1773848 "/etc/varnish/default.vcl"

CLI communication error (hdr)

# reload succeeded by specifying a large timeout.
ubuntu@proxy04:~$ sudo ./varnishreload -t 10
VCL 'reload_20240709_143022_1774368' compiled
VCL 'reload_20240709_143022_1774368' now active
```

### CODE REVIEW

File: systemd/varnishreload [hunk 1]
```diff
@@ -57,6 +58,7 @@ usage() {
 	-n <workdir> : specify the name or directory for the varnishd instance
 	-p <prefix>  : prefix for the name of the loaded and discarded VCLs
 	-w <warmup>  : the number of seconds between load and use operations
+	-t <timeout> : the number of seconds to wait for varnishadm to finish
```
[the remark this answers is not in the document]
Comment (xcir) [reply systemd/varnishreload:60]: Thank you for pointing it out.
I had forgotten add usage.
The order has been corrected as well.

File: systemd/varnishreload [hunk 2]
```diff
@@ -142,6 +144,7 @@ do
 	n) WORK_DIR=$OPTARG ;;
 	p) VCL_PREFIX=$OPTARG ;;
 	w) WARMUP=$OPTARG ;;
+	t) TIMEOUT="-t $OPTARG" ;;
```
[the remark this answers is not in the document]
Comment (xcir) [reply systemd/varnishreload:146]: :)

**Review by xcir — COMMENTED** [review]
Thank you!
fixed.

**Comment by simonvik** [comment]
Looks good to me.

**Pull request merged**